airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/AbstractPostgresSourceDatatypeTest.java

```diff
@@ -847,6 +847,17 @@ private void addArraysTestData() {
             .addExpectedValues("[131070.237689,231072.476596593]")
             .build());
 
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .sourceType("jsonb_array")
+            .fullSourceDataType("JSONB[]")
+            .airbyteType(JsonSchemaType.builder(JsonSchemaPrimitive.ARRAY)
+                .withItems(JsonSchemaType.builder(JsonSchemaPrimitive.STRING).build())
+                .build())
+            .addInsertValues("ARRAY['{\"foo\":\"bar\"}'::JSONB, NULL]")
+            .addExpectedValues("[\"{\\\"foo\\\": \\\"bar\\\"}\",null]")
```

Comment:
This still seem likes Postgres's string representation of the array. I think we want something along the lines of `["{\"foo\": \"bar\"}"]`

Reply:
As discussed offline, disregard this. It's Java's representation of the array. We've tested this version manually and it fixes the regression 👍